iroh-base/Cargo.toml

```diff
@@ -26,6 +26,19 @@ serde = { version = "1", features = ["derive"] }
 serde-error = "0.1.2"
 thiserror = "1"
 
+# key module
+aead = { version = "0.5.2", features = ["bytes"], optional = true }
+derive_more = { version = "1.0.0-beta.1", features = ["debug", "display"], optional = true }
+ed25519-dalek = { version = "2.0.0", features = ["serde", "rand_core"], optional = true }
+once_cell = { version = "1.18.0", optional = true }
+rand = { version = "0.8", optional = true }
+rand_core = { version = "0.6.4", optional = true }
+ssh-key = { version = "0.6.0", features = ["ed25519", "std", "rand_core"], optional = true }
+ttl_cache = { version = "0.5.1", optional = true }
+crypto_box = { version = "0.9.1", features = ["serde", "chacha20"], optional = true }
+zeroize = { version = "1.5", optional = true }
+url = { version = "2.5.0", features = ["serde"], optional = true }
```

Comment:
pretty minor, but i'm not a fan of keeping things in blocks like this.  they start to overlap, get outdated etc.  in the long run a single alphabetically-sorted list of dependencies is almost always nicer (and `cargo add` will keep it sorted, which it can't like this).

Reply:
let's do a followup then that unifies this in the iroh codebase, we have those blocks in a couple of Cargo.tomls in the repo I think.